scripts/objc-test.sh

```diff
@@ -16,9 +16,17 @@ function cleanup {
     WATCHMAN_LOGS=/usr/local/Cellar/watchman/3.1/var/run/watchman/$USER.log
     [ -f $WATCHMAN_LOGS ] && cat $WATCHMAN_LOGS
   fi
+  [ $SERVER_PID ] && kill -9 $SERVER_PID
```

Comment:
That line probably won't do anything because packager is running as a daemon.
For e2e tests we kill all node 0_O, as a temporary measure, of course, until we have a nice nondaemon packager